### DISCUSSION TITLE
Welcome to oak Discussions!

### DISCUSSION THREAD

**Discussion by edgar-manukyan** [pharmaverse/sdtm.oak#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kamilsi** [comment]
Hello everyone,

So excited to be here! My name is Kamil, my involvement with clinical data standards goes back to my time as a contractor at Roche, where I contributed to early versions of the MINT project and also oak. Seeing these projects evolve and now being part of the open-source community as a CDISC volunteer is incredibly rewarding.

I am currently with Transition Technologies Science, where we develop an eCRF platform among other innovative solutions for healthcare. My role here has reinforced my belief in the power of open-source efforts; it's a space where I can contribute meaningfully, ensuring that we're not just beneficiaries of the community's knowledge, but active contributors as well.

I've been working with R since 2006, and it's my tool of choice for data analysis and visualization. I'm looking forward to contributing to the community, particularly by leveraging my R programming expertise to enhance our data standardization efforts.

I'm here to both share my knowledge and learn from all of you. Let's collaborate to make a difference in the way we handle and standardize clinical data!
